src/query/ast/src/parser/token.rs

```diff
@@ -1093,6 +1093,10 @@ pub enum TokenKind {
     RANDOM,
     #[token("IFNULL", ignore(ascii_case))]
     IFNULL,
+    #[token("NVL", ignore(ascii_case))]
```

Comment:
why need to define token?

Reply:
Sorry I got some things wrong, I will remove them, they are not in the keywords.